src/layouts/IndexPage.jsx

```diff
@@ -49,9 +43,49 @@ export const IndexPage = () => {
   const classes = useStyles();
   const [, dispatch] = useContext(GlobalStateContext);
 
+  const [newsList, setNewsList] = useState([]);
+
   useEffect(() => {
-    announce('Pagina caricata ' + name);
+    (async () => {
+      try {
+        await fetch(`https://raw.githubusercontent.com/tuxolino/testapi/main/data/NC002.json`)
```

Comment:
?

Reply:
It's for the notices section, we call the data exposed from salesforce team

Reply:
I updated the url